service/version/metric.go

```diff
@@ -12,13 +12,15 @@ var buildInfo = prometheus.NewGaugeVec(
 		Name:      "build_info",
 		Help:      "A metric with a constant '1' value labeled by commit, golang version, golang os, and golang arch.",
 	},
-	[]string{"commit", "golang_version", "golang_goos", "golang_goarch"},
+	[]string{"bundle_version", "commit", "golang_version", "golang_goos", "golang_goarch"},
```

Comment:
```suggestion
	[]string{"version", "commit", "golang_version", "golang_goos", "golang_goarch"},
```

I would prefer it to call it like that. We need to get rid of bundle concept at some point and I would like to avoid updating alerts when doing so.

Reply:
fine with me